docs/BreakingChanges/141 WPF TreeViewItem must be used within a TreeView.md

```diff
@@ -0,0 +1,27 @@
+## ID: WPF TreeViewItem must be used within a TreeView
+
+### Scope
+Major
```

Comment:
I agree.   See my other comment about the full preconditions to see the bug - they're quite narrow.

Also, if there's some way to indicate to the public that it's just a bug, fixed in 4.6.2 (and in hotfix for 4.5.x, 4.6, 4.6.1).   Not an intentional breaking change.